File: deepseek/nivel basico/c++/suma.cpp
#include <iostream>
using namespace std;

double suma(double a, double b) {
    return a + b;
}

int main() {
    double num1, num2;
    cout << "Ingrese el primer número: ";
    cin >> num1;
    cout << "Ingrese el segundo número: ";
    cin >> num2;
    cout << "La suma es: " << suma(num1, num2) << endl;
    return 0;
}

File: deepseek/nivel basico/ruby/suma.rb
def suma(a, b)
  a + b
end

print "Ingrese el primer número: "
num1 = gets.chomp.to_f
print "Ingrese el segundo número: "
num2 = gets.chomp.to_f
puts "La suma es: #{suma(num1, num2)}"